File: Logic_HandsOn/Java/Problems on Array,Pattern etc/Program91.java
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Write a program which displays ASCII table. 
//		Table contains symbol,	Decimal, Hexadecimal and Octal representation of every member from 0 to 255.
//
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
class Solution
{

	public void DisplayASCII()
	{	
		
		System.out.println("Decimal\t\tOctal\tHexaDecimal\n");
		for(int iCnt=0;iCnt<122;iCnt++)
		{
			System.out.print(iCnt+"\t\t");
			System.out.print((Integer.toOctalString(iCnt))+"\t");
			System.out.print((Integer.toHexString(iCnt))+"\t");
			System.out.println();
		
	
		}
	}


}

class Program1
{
	public static void main(String[] args)
	{

		Solution obj = new Solution();
		obj.DisplayASCII();
	}
}

/*
OUTPUT:
*/

File: Logic_HandsOn/C++/Problems on Digits,String,Array,Pointers etc/program91.cpp
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Write a program which displays ASCII table. 
//		Table contains symbol,	Decimal, Hexadecimal and Octal representation of every member from 0 to 255.
//
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////


#include<iostream>
using namespace std;

class Solution
{
		
	public:
		
		void DisplayASCII()
		{
			int iCnt=0;
			cout<<"Decimal\t\tHexaDecimal\tOctal\n";
			for(iCnt=0;iCnt<122;iCnt++)
			{
				cout<<iCnt<<"\t";
				cout<<hex<<iCnt<<"\t";
				cout<<oct<<iCnt<<"\t";
				cout<<"\n";
			}

		}


};

int main()
{
	
	Solution *dObj = new Solution();
	dObj->DisplayASCII();
	
	delete dObj;
	return 0;
}

/*
OUTPUT:
*/
